### ISSUE TITLE
2.0.2 version import error

### ISSUE THREAD

**Issue by mmyjona** [taoufik07/responder#402]
When impoting responder 2.0.0 to 2.0.2, import error was raised:

> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
>   File "D:\Envs\pp-ng\lib\site-packages\responder\__init__.py", line 1, in <module>
>     from .core import *
>   File "D:\Envs\pp-ng\lib\site-packages\responder\core.py", line 1, in <module>
>     from .api import API
>   File "D:\Envs\pp-ng\lib\site-packages\responder\api.py", line 28, in <module>
>     from .templates import Templates
> ImportError: cannot import name 'Templates' from 'responder.templates' (D:\Envs\pp-ng\lib\site-packages\responder\templates\__init__.py)
> ImportError: cannot import name 'Templates' from 'responder.templates'

Version 1.3.2 imported just fine.

**Comment by mmyjona** [comment]
Also when I test in a new venv. starlette also raised some error.

> ModuleNotFoundError: No module named 'itsdangerous'

**Comment by iancleary** [comment]
I can confirmed both errors:

~~~~
 test_responder pipenv --python 3.7
Creating a virtualenv for this project…
Pipfile: /home/iancleary/dev/test_responder/Pipfile
Using /usr/bin/python3.7 (3.7.5rc1) to create virtualenv…
⠧ Creating virtual environment...Already using interpreter /usr/bin/python3.7
Using base prefix '/usr'
New python executable in /home/iancleary/.local/share/virtualenvs/test_responder-RWmZuL56/bin/python3.7
Also creating executable in /home/iancleary/.local/share/virtualenvs/test_responder-RWmZuL56/bin/python
Installing setuptools, pip, wheel...
done.
Running virtualenv with interpreter /usr/bin/python3.7

✔ Successfully created virtual environment! 
Virtualenv location: /home/iancleary/.local/share/virtualenvs/test_responder-RWmZuL56
Creating a Pipfile for this project…
➜  test_responder ls
Pipfile
➜  test_responder pipenv shell
Launching subshell in virtual environment…
 . /home/iancleary/.local/share/virtualenvs/test_responder-RWmZuL56/bin/activate
➜  test_responder  . /home/iancleary/.local/share/virtualenvs/test_responder-RWmZuL56/bin/activate
(test_responder) ➜  test_responder pipenv install responder
Installing responder…
Adding responder to Pipfile's [packages]…
✔ Installation Succeeded 
Pipfile.lock not found, creating…
Locking [dev-packages] dependencies…
Locking [packages] dependencies…
✔ Success! 
Updated Pipfile.lock (42ca41)!
Installing dependencies from Pipfile.lock (42ca41)…
  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 34/34 — 00:00:03
(test_responder) ➜  test_responder python    
Python 3.7.5rc1 (default, Oct  8 2019, 16:47:45) 
[GCC 9.2.1 20191008] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import responder
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/iancleary/.local/share/virtualenvs/test_responder-RWmZuL56/lib/python3.7/site-packages/responder/__init__.py", line 1, in <module>
    from .core import *
  File "/home/iancleary/.local/share/virtualenvs/test_responder-RWmZuL56/lib/python3.7/site-packages/responder/core.py", line 1, in <module>
    from .api import API
  File "/home/iancleary/.local/share/virtualenvs/test_responder-RWmZuL56/lib/python3.7/site-packages/responder/api.py", line 15, in <module>
    from starlette.middleware.sessions import SessionMiddleware
  File "/home/iancleary/.local/share/virtualenvs/test_responder-RWmZuL56/lib/python3.7/site-packages/starlette/middleware/sessions.py", line 5, in <module>
    import itsdangerous
ModuleNotFoundError: No module named 'itsdangerous'
>>> exit()
(test_responder) ➜  test_responder pipenv install itsdangerous
Installing itsdangerous…
Adding itsdangerous to Pipfile's [packages]…
✔ Installation Succeeded 
Pipfile.lock (6e2a34) out of date, updating to (42ca41)…
Locking [dev-packages] dependencies…
Locking [packages] dependencies…
✔ Success! 
Updated Pipfile.lock (6e2a34)!
Installing dependencies from Pipfile.lock (6e2a34)…
  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 35/35 — 00:00:02
(test_responder) ➜  test_responder python                     
Python 3.7.5rc1 (default, Oct  8 2019, 16:47:45) 
[GCC 9.2.1 20191008] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import responder
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/iancleary/.local/share/virtualenvs/test_responder-RWmZuL56/lib/python3.7/site-packages/responder/__init__.py", line 1, in <module>
    from .core import *
  File "/home/iancleary/.local/share/virtualenvs/test_responder-RWmZuL56/lib/python3.7/site-packages/responder/core.py", line 1, in <module>
    from .api import API
  File "/home/iancleary/.local/share/virtualenvs/test_responder-RWmZuL56/lib/python3.7/site-packages/responder/api.py", line 28, in <module>
    from .templates import Templates
ImportError: cannot import name 'Templates' from 'responder.templates' (/home/iancleary/.local/share/virtualenvs/test_responder-RWmZuL56/lib/python3.7/site-packages/responder/templates/__init__.py)
~~~~

We could add `itsdangerous` to the Pipfile if it's no longer installed by a dependency.

**Comment by vyahello** [comment]
I can confirm same issue on either `v2.0.0` or `v2.0.2`, on `v1.3.2` works well.

**Comment by taoufik07** [comment]
Thanks guys, can you check with `v2.0.3` ?

**Comment by vyahello** [comment]
@taoufik07 it is fixed on `v2.0.3`, awesome, thanks!
